bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -81,7 +79,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
     if (request.getRequestURI().endsWith("profile.jsp")
         || request.getRequestURI().endsWith("register.jsp")
         || request.getRequestURI().endsWith("login.jsp")) {
-      System.out.println("here");
+              System.out.println("redirect to profile");
```

Comment:
Remove :)

Reply:
Removed

Reply:
So you say "removed"... but I still see the println here... (insert suspicious, chin scratching emoji)

Reply:
It says it's removed on my local cloud terminal :/

Reply:
Okay it's gone now